subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/finalizer/verdict/LegacyFailedTestDeterminer.kt

```diff
@@ -0,0 +1,39 @@
+package com.avito.instrumentation.internal.finalizer.verdict
+
+import com.avito.report.model.Flakiness
+import com.avito.report.model.SimpleRunTest
+
+internal class LegacyFailedTestDeterminer(
```

Comment:
Give a brief explanation about the difference between that class and `HasFailedTestDetermainer`